service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerSubmittedTest.java

```diff
@@ -302,34 +277,28 @@ void shouldSendFailedPaymentNotificationOnHiddenDisplayAmountToPay() throws Noti
     }
 
     @Test
-    void shouldNotifyHmctsAdminWhenTheLocalAuthorityHasSubmittedAReturnedCase() throws Exception {
-        given(ldClient.boolVariation(eq("payments"), any(), anyBoolean())).willReturn(true);
+    void shouldNotifyHmctsAdminWhenTheLocalAuthorityHasSubmittedAReturnedCase() {
         CaseDetails caseDetails = enableSendToCtscOnCaseDetails(NO);
         caseDetails.getData().put("displayAmountToPay", NO.getValue());
 
         CallbackRequest callbackRequest = buildCallbackRequest(caseDetails, RETURNED);
         postSubmittedEvent(callbackRequest);
 
-        checkUntil(() -> {
-            verify(notificationClient).sendEmail(
-                eq(AMENDED_APPLICATION_RETURNED_TO_THE_ADMIN),
-                eq(HMCTS_ADMIN_EMAIL),
-                anyMap(),
-                eq(caseId.toString()));
-        });
-
-        checkThat(() -> {
-            verify(notificationClient, never()).sendEmail(
-                eq(AMENDED_APPLICATION_RETURNED_TO_THE_ADMIN),
-                eq(CTSC_EMAIL),
-                anyMap(),
-                eq(caseId.toString()));
-        });
+        checkUntil(() -> verify(notificationClient).sendEmail(
+            eq(AMENDED_APPLICATION_RETURNED_TO_THE_ADMIN),
+            eq(HMCTS_ADMIN_EMAIL),
+            anyMap(),
+            eq(caseId.toString())));
+
+        checkThat(() -> verify(notificationClient, never()).sendEmail(
+            eq(AMENDED_APPLICATION_RETURNED_TO_THE_ADMIN),
+            eq(CTSC_EMAIL),
+            anyMap(),
+            eq(caseId.toString())));
     }
 
     @Test
-    void shouldNotifyCtscAdminWhenTheLocalAuthorityHasSubmittedAReturnedCase() throws Exception {
-        given(ldClient.boolVariation(eq("payments"), any(), anyBoolean())).willReturn(true);
+    void shouldNotifyCtscAdminWhenTheLocalAuthorityHasSubmittedAReturnedCase() {
         CaseDetails caseDetails = enableSendToCtscOnCaseDetails(YES);
         caseDetails.getData().put("displayAmountToPay", NO.getValue());
```

Comment:
How about making converting them to static final String? ("displayAmountToPay", "payments"    etc)

Reply:
moved "displayAmountToPay" to constant.